test/src/main/java/org/apache/accumulo/test/functional/MemoryStarvedScanIT.java

```diff
@@ -162,13 +173,11 @@ private void consumeServerMemory(BatchScanner scanner) {
     assertTrue(iter.hasNext());
   }
 
-  static void freeServerMemory(AccumuloClient client, String table) throws Exception {
-    try (Scanner scanner = client.createScanner(table)) {
-      scanner.addScanIterator(new IteratorSetting(11, MemoryFreeingIterator.class, Map.of()));
-      @SuppressWarnings("unused")
-      Iterator<Entry<Key,Value>> iter = scanner.iterator(); // init'ing the iterator should be
-                                                            // enough to free the memory
-    }
+  static void freeServerMemory(AccumuloClient client) throws Exception {
+    // Instantiating this class on the TabletServer will free the memory as it
+    // frees the buffers created by the MemoryConsumingIterator in its constructor.
+    client.instanceOperations().testClassLoad(MemoryFreeingIterator.class.getName(),
+        WrappingIterator.class.getCanonicalName());
```

Comment:
In this case, it doesn't matter, but you can't use canonical name to specify the type in the general case.

```suggestion
        WrappingIterator.class.getName());
```

Reply:
Addressed in 55265e8199